source/crud/bulk-write.md

```diff
@@ -922,8 +906,19 @@ number was determined by constructing `OP_MSG` messages with various fields atta
 Drivers are required to use this value even if they are capable of determining the exact size of the message prior to
 batch-splitting to standardize implementations across drivers and simplify batch-splitting testing.
 
+### Why is there no requirement to validate the size of a BSON document?
```

Comment:
Suggest noting a previous version of the spec included size checks. That may help describe why this rationale is here:

```md
Following ["_Where possible, depend on server to return
errors_"](https://github.com/mongodb/specifications/blob/f8dbd2469f18d093f917efa1f758024bca5d3aaa/source/driver-mantras.md#where-possible-depend-on-server-to-return-errors),
drivers should rely on the server to return errors. Such reliance is not possible for unacknowledged writes.

This specification previously required drivers to check size limits for unacknowledged writes. This requirement has
since been removed. Checking size limits complicates some driver implementations. Returning a driver error in this
specific situation does not seem helpful enough to require size checks.
```

Reply:
Done in f90c80762b3a9a9ea4b98b36275b5d8f7f29147e.